cryptocurrency-subdomain/src/main/kotlin/com/ajax/cryptocurrency/application/ports/kafka/KafkaConsumer.kt

```diff
@@ -0,0 +1,5 @@
+package com.ajax.cryptocurrency.application.ports.kafka
```

Comment:
don't bind infrastructure components naming or packages to your application level

and this exact interface is not needed probably

Reply:
on 1-2-1 you said that its better to make interfaces for kafka producer/consumer, but actually I can remove them

Reply:
interfaces removed